ql/src/java/org/apache/hadoop/hive/ql/exec/repl/util/ReplUtils.java

```diff
@@ -219,16 +224,16 @@
     return TaskFactory.get(replLogWork, conf);
   }
 
+  public static boolean isDbBeingFailedOverAtSource(Database db) {
+    assert (db != null);
```

Comment:
Does this hold good for 'REPL DUMP *' use case?

Reply:
For repl dump * , execution would not come here ever.